magpie/api/management/user/user_utils.py

```diff
@@ -106,10 +112,42 @@ def _add_to_group(usr, grp):
         _add_to_group(new_user, _get_group(anonym_grp_name))
         new_user_groups.append(anonym_grp_name)
 
+    # Check for webhook requests
+    config_path = get_constant("MAGPIE_CONFIG_PATH", default_value=None,
+                               raise_missing=False, raise_not_set=False, print_missing=True)
+    if config_path:
+        webhook_configs = get_all_configs(config_path, "webhooks", allow_missing=True)
+        for cfg in webhook_configs:
+            if "create" in cfg.keys() and len(cfg["create"]) > 0:
+                # Execute all webhook requests
+                pool = multiprocessing.Pool(processes=len(cfg["create"]))
+                args = [(url, user_name) for url in cfg["create"]]
+                pool.starmap_async(webhook_request, args, error_callback=webhook_error_callback)
+
     return ax.valid_http(http_success=HTTPCreated, detail=s.Users_POST_CreatedResponseSchema.description,
                          content={"user": uf.format_user(new_user, new_user_groups)})
 
 
+def webhook_request(webhook_url, user_name):
```

Comment:
I can see the webhook operations applied to more than just users. Both `webhook_request` and `webhook_error_callback` should be moved to `magpie.api.requests` module.